app/assets/javascripts/listings/components/browseListings.js.coffee

```diff
@@ -28,5 +29,19 @@ angular.module('dahlia.components')
       else if @tenureType == 'rental'
         $translate.instant('listings.you_dont_match_any_rent')
 
+    @resultsHeader = () ->
```

Comment:
I'm having a hard time parsing all of these different pieces of logic. I think we still want to combine it into one big sentence for all of the different potential outputs. 

Can you list out what all of the different final possibilities are?

Reply:
Sure, there are 12 different options here. There are always one or more household members with income calculated per month or year. That are 4 options. Of each of those we can have no child, one or multiple.

Reply:
OK I asked a couple of folks who speak chinese here in the office and they think that it can be OK as you have it (pending update to move from "/month" to "per month").

I'll work with them directly to make sure that the translations end up making sense in the end.